### DISCUSSION TITLE
Kunena : Login with Google | Facebool | Linkedlin

### DISCUSSION THREAD

**Discussion by jcosmesal** [Kunena/Kunena-Forum#9753] [Ideas]
Dear community.
:-)  It is necessary for Kunena to have a login with the most active and popular platforms. Please support me to start this project. I am waiting for you to tell me how I can contribute.

**Comment by Ruud68** [comment]
Hi, Kunena uses Joomla user account creation and login logic. Nothing more, nothing less.
If you implement e.g. this: https://github.com/akeeba/sociallogin then you extend Joomla with linkedIn, facebook, etc. social logins and by doing that you can also use that for Kunena

**Reply by jcosmesal** [reply]
Excelent . this is great
